imod/formats/prj/prj.py

```diff
@@ -522,30 +522,63 @@ def _try_read_with_func(func, path, *args, **kwargs):
         raise type(e)(f"{e}. Error thrown while opening file: {path}")
 
 
-def _create_datarray_from_paths(paths: List[str], headers: List[Dict[str, Any]]):
+def _get_array_transformation_parameters(
+    headers: List[Dict[str, Any]], key: str, dim: str
+) -> xr.DataArray:
+    if dim in headers[0].keys():
```

Comment:
Maybe add a short docstring what this helper function does?

Reply:
done